embark.el

```diff
@@ -3638,8 +3646,12 @@ and leaves the point to the left of it."
   ("5" embark-hash-sha512)
   ("f" format-encode-region)
   ("F" format-decode-region)
+  ("b" base64-encode-region)
+  ("B" base64-decode-region)
   ("u" embark-encode-url)
-  ("U" embark-decode-url))
+  ("U" embark-decode-url)
+  ("c" epa-encrypt-region)
+  ("C" embark-epa-decrypt-region))
```

Comment:
I'm not familiar with these functions: why is `epa-encrypt-region` OK as-is but `epa-decrypt-region` requires a little wrapper?

Reply:
It is an inconsistency in Emacs. encrypt always replaces but decrypt asks by default :shrug: